docs/topics/mta-web-proxy-config.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * docs/cli-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-proxy-config_{context}"]
+= Configuring HTTP and HTTPS proxy settings
+
+This management module configures HTTP & HTTPS proxy settings.
+
+.Procedure
+
+. Click the radio button to enter the following information.
```

Comment:
Which radio button? Start with the next sentence (delete this).

Reply:
Deleted